__tests__/api.test.js

```diff
@@ -136,4 +144,24 @@ describe("API tests", ()=>{
             })
         })
     })
+    describe("DELETE /api/comments/:comment_id", () => {
+        test("204: Deletes a comment by the comment ID", () => {
+            return request(app)
+            .delete("/api/comments/1")
+            .expect(204)
+        })
+        test("404: Check the comment was successfully deleted", () => {
+            return request(app)
+            .get(`/api/comments/1`)
+            .expect(404)
+        })
+        test("404: Returns correct error if there was no comment to delete", () => {
+            return request(app)
+            .delete("/api/comments/150")
+            .expect(404)
+            .then(response => {
+                expect({ msg: "No comment to delete"})
```

Comment:
You would probably want something like id doesn't exist. Something more generic so you can use that error handler for any time the id doesn't exist.